anasbinarif/ace-my-exam
# Hotfixes for Admin

- Change Entrance exam Age groups to 11+, 13+, etc...
- Remove AQA from IGCSE
- make sure that subject subtype clears on change
- textual changes
- show and hide states for past_paper, topical_questions and revision_notes options
- Add chevrons to dropdowns
- Remove bolding of chapter and topic
- make larger file uploads and error handling on FE
- FIX: uploading pastpaper linked to subject
